viz_scripts/bin/generate_plots.py

```diff
@@ -121,9 +121,9 @@ def compute_for_date(month, year):
     # Execute the notebook with the new parameters
     nbclient.execute(new_nb)
 
-# Compute the overall metrics
-compute_for_date(None, None)
-
 # Compute for every month until now
 for month_year in compute_range:
     compute_for_date(month_year.month, month_year.year)
+
+# Compute the overall metrics
+compute_for_date(None, None)
```

Comment:
Given that we throw the exception from the notebook, I don't think that moving this is strictly required. But it doesn't hurt, so I am going to go ahead and merge it.